packages/enzyme-adapter-react-14/src/ReactFourteenAdapter.js

```diff
@@ -23,7 +23,9 @@ import {
   getNodeFromRootFinder,
   wrapWithWrappingComponent,
   getWrappingComponentMountRenderer,
+  spyMethod,
 } from 'enzyme-adapter-utils';
```

Comment:
I ended up adding spyMethod/spyProperty to enzyme-adapter-utils, so that the adapters didn't have to suddenly start requiring enzyme v3.4.0.